ui/src/components/ensembler/StandardEnsemblerLoaderComponent.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+
+import useDynamicScript from "../../hooks/useDynamicScript";
+
+// Renderless component wrapper
+const LoadDynamicScript = ({ url, setReady, setFailed }) => {
+  const { ready, failed } = useDynamicScript({
+    url: url,
+  });
+
+  useEffect(() => {
+    setReady(ready);
+    setFailed(failed);
+  }, [setReady, setFailed, ready, failed]);
+
+  return null;
+};
+
+// Dynamic Script Loading component wrapper
+export const StandardEnsemblerLoaderComponent = ({
```

Comment:
I think there's room to combine this component with the existing `ExperimentEngineLoaderComponent`. The only real difference is the props naming for `remoteUi` vs. `experimentEngine`, even the `LoadDynamicScript` is the same. Perhaps, we can shift this file to `src/components/remote_component` now that it is used in both ensembler and experiments' related components?

Reply:
Okay done, now both of those components got reduce to a single `RemoteLoaderComponent` :)